webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -8778,6 +8788,7 @@ var CURRENT_GROUP_DATA={};
 
                 var displayName = mckMessageLayout.getTabDisplayName(contact.contactId, isGroup);
                 var notificationsound = mckNotificationTone;
+                KommunicateUI.togglePopupChatTemplate();
```

Comment:
@CaptainT33mo which is correct clearMsgTriggerAndChatPopuTimeouts or  KommunicateUI.togglePopupChatTemplate();?

Reply:
@devashishmamgain `KommunicateUI.togglePopupChatTemplate();` and replaced it with `mckInit.clearMsgTriggerAndChatPopuTimeouts();`